www/js/main.js

```diff
@@ -50,19 +40,320 @@ angular.module('emission.main', ['emission.main.diary', 'emission.main.common',
         controller: 'ControlCtrl'
       }
     }
+  })
+  .state('root.main.sensed', {
```

Comment:
it is not wrong to have the states here, but it is not strictly necessary.
you can navigate to the root.main.recent.sensed state from the main template.